integration/env.rs

```diff
@@ -2,16 +2,21 @@ use std::env;
 
 use birdcage::{Birdcage, Exception, Sandbox};
 
-fn main() {
+use crate::TestSetup;
+
+pub fn setup() -> TestSetup {
     // Setup our environment variables
     env::set_var("PUBLIC", "GOOD");
     env::set_var("PRIVATE", "BAD");
 
     // Activate our sandbox.
-    let mut birdcage = Birdcage::new();
-    birdcage.add_exception(Exception::Environment("PUBLIC".into())).unwrap();
-    birdcage.lock().unwrap();
+    let mut sandbox = Birdcage::new();
+    sandbox.add_exception(Exception::Environment("PUBLIC".into())).unwrap();
+
+    TestSetup { sandbox, data: String::new() }
+}
 
+pub fn validate(_data: String) {
```

Comment:
Unrelated to this PR (which looks good!), but this could test that no other env vars are present (i.e. test the length of `env::vars`). I was a bit surprised that on Mac some env var is added by the OS when spawning with `process::Command`, that must additionally be cleared on the child-side. (Not sure if it's an actual security threat though.)

Reply:
Yeah that's a good point. We control the full environment so it should be no problem verifying all of it.